### DISCUSSION TITLE
Scroll position no longer resets after using back button

### DISCUSSION THREAD

**Discussion by arp242** [community/community#21201] [Other Feature Feedback, Questions, & Ideas] [answered]
After using the back button, the scroll position no longer gets reset when loading new pages. For example:

1. Load https://github.com/fsnotify/fsnotify/issues in a new tab.
2. Scroll down a bit.
3. Click any issue: this loads fine.
4. Go back in the browser, returning to the issue list.
5. Click another issue: the scroll position remains identical to the scroll position of the issue list, rather than starting from the top.
6. This tab is now borked and it will happen with many links.

This is not just with issues; it happens with many (all?) pages; for example:

- Selecting the next page in discussions.
- Clicking "issues" or something else at the top.
- Opening a file in the code browser.
- etc.

Tested with Firefox 102 and Chromium 103.0.5060.53 with a new empty profile, both on Linux. Both have the same behaviour. Happens both when logged in and logged out.

I don't know if this is the best place to report bugs, but it's the only thing I could find.

**Comment by radar-blip** [comment]
@arp242, This has been happening often lately, I'm sure staff is aware of it but you can help speed things up by filing a bug report.
You can do so here: 
https://support.github.com/contact/bug-report

**Reply by arp242** [reply]
Cheers, I sent it there too.

**Comment by arp242** [comment, marked as answer]
It's fixed now.
